site/app/libraries/database/DatabaseQueries.php

```diff
@@ -3331,6 +3331,21 @@ public function getRegistrationSections() {
         return $this->course_db->rows();
     }
 
+    /**
+     * Gets the default registration section for a given course and term from the courses table
+     */
+    public function getDefaultRegistrationSection(string $term, string $course): string {
+        $this->submitty_db->query("SELECT default_section_id FROM courses where term=? and course=?", [$term, $course]);
+        return $this->submitty_db->row()['default_section_id'];
+    }
```

Comment:
Is this function necessary?  Is it possible to get the same data by adding this field to an existing query which is already being run to fetch the data for a course?

Reply:
I don't see a function that I can add this to, most of the data that is being searched for is stored in different databases, and it would probably be confusing to join two separate queries.